docs/networking.md

```diff
@@ -28,6 +28,16 @@ Unlike for [TCP protocol](#tcp-configuration), there is no separation between in
 
 When the `ExtIp` setting is not provided, EventStoreDB will use the first available non-loopback address. You can also bind HTTP to all available interfaces using `0.0.0.0` as the setting value. If you do that, you'd need to configure the `ExtHostAdvertiseAs` setting (read mode [here](#network-address-translation)), since `0.0.0.0` is not a valid IP address to connect from the outside world.
 
+::: warning
```

Comment:
Can you update the documentation to always reference the new options rather than the deprecated ones (e.g. use `NodeIp` rather than `ExtIp` in the above text), and replace the format/syntax tables with the new options as well.

I don't think we need to keep the format/syntax tables of the deprecated options. The deprecation warning should be enough.

Reply:
Done